test/powershell/Language/Scripting/OutErrorVariable.Tests.ps1

```diff
@@ -75,6 +75,48 @@ Describe "Tests OutVariable only" -Tags "CI" {
         $script:a | Should Be 'foo'
         $b | Should Be @("bar", "foo")
     }
+
+    $outVarTypeTestData = @(
```

Comment:
It is used once - please move directly in `-TestCases`.

Reply:
Happy to do this. Is there an example of the code style you can link where `$TestCases` is inlined in the call?